fastlane/Fastfile

```diff
@@ -432,29 +436,28 @@ end
       api_key_path: File.join(Dir.home, '.configure', 'simplenote-ios', 'secrets', 'app_store_connect_fastlane_api_key.json')
     )
 
-    sh("rm ../Simplenote.ipa")
-    dSYM_PATH = File.dirname(Dir.pwd) + "/Simplenote.app.dSYM.zip"
+    sh('rm ../Simplenote.ipa')
+    dSYM_PATH = "#{File.dirname(Dir.pwd)}/Simplenote.app.dSYM.zip"
 
     sentry_upload_dsym(
       dsym_path: dSYM_PATH,
-      auth_token: ENV["SENTRY_AUTH_TOKEN"],
+      auth_token: ENV.fetch('SENTRY_AUTH_TOKEN', nil),
       org_slug: 'a8c',
-      project_slug: 'simplenote-ios',
+      project_slug: 'simplenote-ios'
     )
 
     sh("rm #{dSYM_PATH}")
 
-    if (options[:create_release])
-      archive_zip_path = File.dirname(Dir.pwd) + "/Simplenote.xarchive.zip"
+    if options[:create_release]
+      archive_zip_path = "#{File.dirname(Dir.pwd)}/Simplenote.xarchive.zip"
       zip(path: lane_context[SharedValues::XCODEBUILD_ARCHIVE], output_path: archive_zip_path)
 
       version = options[:beta_release] ? ios_get_build_version : ios_get_app_version
-      create_release(repository:GHHELPER_REPO,
-        version: version,
-        release_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'Simplenote', 'Resources', 'release_notes.txt'),
-        release_assets: archive_zip_path.to_s,
-        prerelease: options[:beta_release]
-      )
+      create_release(repository: GHHELPER_REPO,
+                     version: version,
+                     release_notes_file_path: File.join(PROJECT_ROOT_FOLDER, 'Simplenote', 'Resources', 'release_notes.txt'),
+                     release_assets: archive_zip_path.to_s,
+                     prerelease: options[:beta_release])
```

Comment:
Same re: newline after opening parens for reduced indentation level

Reply:
Fixed in https://github.com/Automattic/simplenote-ios/commit/a15e5d22d216e41b22583777cdb3e5ce79235de1